docs/reference/search/search-your-data/ccs-version-compat-matrix.asciidoc

```diff
@@ -0,0 +1,12 @@
+[cols="^,^,^,^,^,^,^"]
+|====
+| 6+^h| Remote cluster version
+h| Local cluster version
+                  |  6.8        | 7.1–7.16   | 7.17       | 8.0        | 8.1        | 8.2
+| 6.8             |  {yes-icon} | {yes-icon} | {yes-icon} | {no-icon}  | {no-icon}  | {no-icon}
+| 7.1–7.16        |  {yes-icon} | {yes-icon} | {yes-icon} | {no-icon}  | {no-icon}  | {no-icon}
+| 7.17            |  {yes-icon} | {yes-icon} | {yes-icon} | {yes-icon} | {yes-icon} | {yes-icon}
+| 8.0             |  {no-icon}  | {no-icon}  | {yes-icon} | {yes-icon} | {yes-icon} | {yes-icon}
+| 8.1             |  {no-icon}  | {no-icon}  | {no-icon}  | {yes-icon} | {yes-icon} | {yes-icon}
+| 8.2             |  {no-icon}  | {no-icon}  | {no-icon}  | {no-icon}  | {yes-icon} | {yes-icon}
+|====
```

Comment:
I get confused: didn't we have a similar table somewhere? I would expect the old one to be replaced with the updated one but I only see the new one being added. What am I missing?

Reply:
The current docs reuse the compatibility table from the [remote clusters docs](https://www.elastic.co/guide/en/elasticsearch/reference/master/remote-clusters.html#sniff-mode).

Since those docs still use that table, I'm not removing it with this PR.